modules/source-bookshelf/lib/process_sort.js

```diff
@@ -3,11 +3,12 @@ const _ = require('lodash');
 // TODO: investigate how to deal with express's query parser
 // converting + into a space.
 function isAscending (key) {
-  return key[0] === '+' || key[0] === ' ';
+  return key[0] === '+' || key[0] === '-';
 }
 
 module.exports = function (model, query, sortBy) {
   var validFields = model.fields;
+  console.log(model.fields);
```

Comment:
....but that is breaking down here because the model isn't wrapped by the source. I can't look now, but we should probably just be sending in the source, not the model itself.